package.json

```diff
@@ -32,6 +32,7 @@
     "cf-pagination": "^4.2.2",
     "cf-tables": "^4.2.1",
     "cf-typography": "^4.6.0",
+    "grunt-contrib-uglify-es": "^3.3.0",
```

Comment:
This should replace `grunt-contrib-uglify` in the `devDependencies` list, and `uglify-es` can also be removed now.